backend/geohashing.js

```diff
@@ -47,69 +30,3 @@ function encodeGeohash(lat, long, precision = 12) {
     }
     return geohash;
 }
-
-function geohashes(coordinates, min_long, max_long, min_lat, max_lat, geohash) {
-    if (coordinates.length < 100) {
-        return coordinates.map(coord => {
-            const lat = coord.latitude;
-            const long = coord.longitude;
-            const geoHash = encodeGeohash(lat, long);
-            return { ...coord, geohash: geoHash };
-        });
-    } else {
-        return subdivide(coordinates, min_long, max_long, min_lat, max_lat, geohash);
-    }
-}
-
-function subdivide(coordinates, min_long, max_long, min_lat, max_lat, geohash) {
-    const midLat = (min_lat + max_lat) / 2;
-    const midLong = (min_long + max_long) / 2;
-
-    const topLeft = coordinates.filter(c => c.latitude >= midLat && c.longitude < midLong);
-    const topLeftResult = geohashes(topLeft, min_long, midLong, midLat, max_lat, geohash + '0');
-
-    const topRight = coordinates.filter(c => c.latitude >= midLat && c.longitude >= midLong);
-    const topRightResult = geohashes(topRight, midLong, max_long, midLat, max_lat, geohash + '1');
-
-    const bottomLeft = coordinates.filter(c => c.latitude < midLat && c.longitude < midLong);
-    const bottomLeftResult = geohashes(bottomLeft, min_long, midLong, min_lat, midLat, geohash + '2');
-
-    const bottomRight = coordinates.filter(c => c.latitude < midLat && c.longitude >= midLong);
-    const bottomRightResult = geohashes(bottomRight, midLong, max_long, min_lat, midLat, geohash + '3');
-
-    return [
-        ...topLeftResult,
-        ...topRightResult,
-        ...bottomLeftResult,
-        ...bottomRightResult,
-    ];
-}
```

Comment:
Hmm, why are we removing this logic? Let's keep it in here, because eventually we will need it to create the original geohashes.

Reply:
Noted. I will create a new file for the pseudocode